projects/ngrx.io/content/guide/store/reducers.md

```diff
@@ -69,34 +69,12 @@ The initial values for the `home` and `away` properties of the state are 0.
 The reducer function's responsibility is to handle the state transitions in an immutable way. Define a reducer function that handles the actions for managing the state of the scoreboard.
 
 <code-example header="scoreboard.reducer.ts">
-export function reducer(
-  state = initialState,
-  action: ScoreboardPageActions.ActionsUnion
-): State {
-  switch (action.type) {
-    case ScoreboardPageActions.homeScore.type: {
-      return {
-        ...state,
-        home: state.home + 1,
-      };
-    }
-
-    case ScoreboardPageActions.awayScore.type: {
-      return {
-        ...state,
-        away: state.away + 1,
-      };
-    }
-
-    case ScoreboardPageActions.reset.type: {
-      return { home: action.home, away: action.away };
-    }
-
-    default: {
-      return state;
-    }
-  }
-}
+export const reducer = createReducer(
+  initialScoreState,
+  on(ScoreboardPageActions.homeScore, (state): State => ({ ...state, home: state.home + 1 })),
```

Comment:
State should infer, we don't have to type it. 

```suggestion
  on(ScoreboardPageActions.homeScore, (state) => ({ ...state, home: state.home + 1 })),
```

(Same for the `on`s below)